src/types/chart/funnel_chart.ts

```diff
@@ -0,0 +1,26 @@
+import { ChartConfiguration } from "chart.js";
+import { Color } from "../misc";
+import { AxesDesign, CustomizedDataSet, TitleDesign } from "./chart";
+import { LegendPosition } from "./common_chart";
+
+export interface FunnelChartDefinition {
+  readonly type: "funnel";
+  readonly dataSets: CustomizedDataSet[];
+  readonly dataSetsHaveTitle: boolean;
+  readonly labelRange?: string;
+  readonly title: TitleDesign;
+  readonly background?: Color;
+  readonly legendPosition: LegendPosition;
+  readonly horizontal?: boolean;
+  readonly axesDesign?: AxesDesign;
+  readonly aggregated?: boolean;
+  readonly showValues?: boolean;
+  readonly funnelColors?: FunnelChartColors;
+}
+
+export type FunnelChartRuntime = {
+  chartJsConfig: ChartConfiguration;
+  background: Color;
+};
+
+export type FunnelChartColors = (Color | undefined)[];
```

Comment:
in the command, I see `null` in the logs, maybe it's because it's been stringified